staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -72,8 +72,12 @@ var (
 		prometheus.HistogramOpts{
 			Name: "apiserver_request_latency_seconds",
 			Help: "Response latency distribution in seconds for each verb, group, version, resource, subresource, scope and component.",
+			// This metric is used for verifying api call latencies SLO,
```

Comment:
We can do that now, since this metric was added in this release.

Just wanted to clarify one more thing - how I can get histogram of values from say last 5 minutes?

Reply:
To get the 99th percentile over the past 5min of data, the query in prometheus would be: 

histogram_quantile(0.99, rate(apiserver_request_latency_seconds_bucket[5m]))

Reply:
@brancz `histogram_quantile(0.99, rate(apiserver_request_latency_seconds_bucket[5m]))` is something that can be calculated by the server, right? The metric endpoint that was discussed here provides only values for the buckets, not a time series?

Reply:
It is calculated by the server, correct. Each bucket is a counter, so each bucket is 1 time-series ingested by Prometheus.